src/api/StringLatinModelService.ts

```
@@ -100,12 +106,14 @@ class StringLatinModelServiceImpl implements IStringLatinModelService {
 
   private getKeyboardData(keyboardId: string): Character[] {
     switch (keyboardId) {
-      case 'characters-DIN-SPEC-91379':
+      case "characters-DIN-SPEC-91379":
         return (characterDataDIN as CharacterData).characters;
-      case 'characters-german':
+      case "characters-german":
         return (characterDataGerman as CharacterData).characters;
       default:
-        console.warn(`Unknown keyboard ID: ${keyboardId}, falling back to DIN keyboard`);
+        console.warn(
```

ESLint

[error] Unexpected console statement. Only these console methods are allowed: debug.

(no-console)